src/channel_state.ts

```diff
@@ -437,6 +437,51 @@ export class ChannelState<
     return messageWithReaction;
   }
 
+  removeQuotedMessages(
```

Comment:
I think should be more like `updateQuotedMessages`, and it should also be called on `message.updated` event. Right?

Reply:
Or maybe we need another function called `updateQuotedMessages`, since there some functionality specific to removing/cleaning quoted message in this function.

Reply:
Also can you add unit test for this function?

Reply:
yeah tbh this function could be used for updating as well but the main problem is that it wasnt erasing the attachments so we had to add that empty array on attachments.

lets add the update in another pr once i had some trouble with this remove logics and dont really want to have the same kind of problems in the update before moving the sprint card lol